### ISSUE TITLE
Failed parsing HTTP/2 and Endpoint read failed

### ISSUE THREAD

**Issue by mortenmoulder** [googlesamples/assistant-sdk-python#3]
I'm getting two errors after a while.

- Freshly installed Raspberry Pi 2 Model B
- Running Python 3
- Hooked up with Ethernet
- Using USB headset

**Endpoint read failed**
```
Traceback (most recent call last):
  File "/usr/lib/python3.4/runpy.py", line 170, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.4/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/pi/env/lib/python3.4/site-packages/googlesamples/assistant/__main__.py", line 273, in <module>
    main()
  File "/home/pi/env/lib/python3.4/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/pi/env/lib/python3.4/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/pi/env/lib/python3.4/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/pi/env/lib/python3.4/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/pi/env/lib/python3.4/site-packages/googlesamples/assistant/__main__.py", line 234, in main
    grpc_deadline):
  File "/home/pi/env/lib/python3.4/site-packages/grpc/_channel.py", line 363, in __next__
    return self._next()
  File "/home/pi/env/lib/python3.4/site-packages/grpc/_channel.py", line 346, in _next
    raise self
grpc._channel._Rendezvous: <_Rendezvous of RPC that terminated with (StatusCode.UNAVAILABLE, Endpoint read failed)>
```

**Failed parsing HTTP/2**
```
Traceback (most recent call last):
  File "/usr/lib/python3.4/runpy.py", line 170, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.4/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/pi/env/lib/python3.4/site-packages/googlesamples/assistant/__main__.py", line 273, in <module>
    main()
  File "/home/pi/env/lib/python3.4/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/pi/env/lib/python3.4/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/pi/env/lib/python3.4/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/pi/env/lib/python3.4/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/pi/env/lib/python3.4/site-packages/googlesamples/assistant/__main__.py", line 234, in main
    grpc_deadline):
  File "/home/pi/env/lib/python3.4/site-packages/grpc/_channel.py", line 363, in __next__
    return self._next()
  File "/home/pi/env/lib/python3.4/site-packages/grpc/_channel.py", line 346, in _next
    raise self
grpc._channel._Rendezvous: <_Rendezvous of RPC that terminated with (StatusCode.UNAVAILABLE, Failed parsing HTTP/2)>
```

**Comment by proppy** [comment]
Yep, I believe this is related to #2

**Comment by proppy** [comment]
Merging with #2

**Comment by proppy** [comment]
Should be fixed with 7da41cf, feel free to give it a try
